### DISCUSSION TITLE
How to enable vulnerability svg badges?

### DISCUSSION THREAD

**Discussion by admantium-sg** [DependencyTrack/dependency-track#2941] [Q&A] [answered]
Hello,

I would like to render vulnerability badges for projects scanned by dependency check.

Following the general documentation (https://docs.dependencytrack.org/integrations/badges/) and the API spec (https://docs.dependencytrack.org/integrations/rest-api/), I could create a correct request, but its return code is 204 "Badge support is disabled. No content will be returned.".

However, neither the general documentation or the configuration documentation (https://docs.dependencytrack.org/getting-started/configuration/) mentions a suitable property.

Digging into the source code, I found this: https://github.com/DependencyTrack/dependency-track/blob/58a83978f714d5940ef7f35cc386b255cbd510f7/src/main/java/org/dependencytrack/resources/v1/BadgeResource.java#L57

How can I enable badge rendering?

Kind regards,

Sebastian

**Comment by syalioune** [comment, marked as answer]
Hello @admantium-sg ,
You can use the `POST /v1/configProperty` endpoint to update the property value.

**Reply by GuilleFB** [reply]
Another option is: Inside Dependency Track frontend if you go to `Administration -> Configuration -> General` you have a tab under `Base URL` to enable the SVG badges

**Comment by admantium-sg** [comment]
Thank you very much!
